packages/xo-web/src/xo-app/settings/users/index.js

```diff
@@ -78,6 +80,15 @@ const USER_COLUMNS = [
     itemRenderer: user =>
       isEmpty(user.authProviders) && <Editable.Password onChange={password => editUser(user, { password })} value='' />,
   },
+  {
+    name: 'OTP',
+    itemRenderer: user =>
```

Comment:
I think it's ok in a separate column as it can also be used to see which users are using OTP. But in that case, for users who don't use OTP, it should say something like "Not configured" or "Disabled".